src/mesh/test/tstDraco_Mesh.cc

```diff
@@ -65,6 +65,12 @@ void cartesian_mesh_2d(rtt_c4::ParallelUnitTest &ut) {
   if (mesh->get_cell_to_node_linkage() != cell_to_node_linkage)
     ITFAILS;
 
+  // check that flat side type and side to node linkage is correct
```

Comment:
Optionally - consider using `FAIL_IF_NOT(mesh->get_side_node_count() == side_node_count);`

Reply:
Thank you for the recommendation - I will plan to change these to `FAIL_IF_NOT`.